ui/tests/test_ui_multi_cluster.py

```
@@ -0,0 +1,338 @@
+    """Switching to a different environment must reset the namespace chip selection."""
+    with sync_playwright() as ctx:
+        browser = ctx.chromium.launch(headless=True)
+        page = browser.new_page()
+        _open_page(page, MULTI_CLUSTER_CONFIG)
+
+        # Select a namespace in the current environment
+        _select_namespace(page, 0)
+        page.wait_for_selector(".v-autocomplete .v-chip__content", timeout=10000)
+        assert len(page.query_selector_all(".v-autocomplete .v-chip__content")) == 1
+
+        # Switch environment – this triggers fetchNamespacesData which resets namespaceName
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)